src/prefect/schedules/clocks.py

```diff
@@ -248,11 +253,13 @@ def __init__(
         end_date: datetime = None,
         parameter_defaults: dict = None,
         labels: List[str] = None,
+        day_or: bool = None,
     ):
         # build cron object to check the cron string - will raise an error if it's invalid
         if not croniter.is_valid(cron):
             raise ValueError("Invalid cron string: {}".format(cron))
         self.cron = cron
+        self.day_or = day_or
```

Comment:
We'll want to default to `True` to match the existing behavior. Right now with it set to `None` croniter will interpret that as `False`. We want to do this within the code (rather than as the default kwarg) so that our serializers do the right thing if the value is missing.

```suggestion
        self.day_or = True if day_or is None else day_or
```